docs/FAQ.md

```diff
@@ -33,6 +33,29 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### `jj` is said to record the working after `jj log` and every other command. Where can I see these automatic "saves"?  
+
+Indeed, every `jj` command updates the current "working-copy" revision, marked 
+with `@` in `jj log`. You can notice this by how the [commit id] of the
+working copy revision changes when it's updated. Note that, unless you move to
+another revision (with `jj new` or `jj edit`, for example), the [change id] will 
+not change.
+
+If you expected to see a historical view of your working-copy changes in 
+`jj log`, as a chain in a parent-child relationship, this is not the case. 
+Instead each commit gets amended and the commit-id changes.
+
+You can see the history of these changes using `jj obslog`. This will show the 
+history of the commits that were previously the "working-copy commit", since 
```

Comment:
I'm guessing the quotes around "working-copy" before was because you introduced the term there. There's no need to quote again here (and below) then, right? Maybe it's just me, but the repeated quotes feel like scare quotes.

Reply:
the quotes stem from ilya's suggestion, I just made the spelling consistent. They're removed now.